builder/Dockerfile

```diff
@@ -77,8 +79,11 @@ ADD pkg pkg
 ADD vendor vendor
 ADD go.mod go.mod
 ADD go.sum go.sum
-RUN mkdir -p /go/bin && \
-	time go build -mod=vendor -o /go/bin/ ./cmd/...
+
+RUN --mount=type=cache,target=/root/.cache/go-build \
+    GOOS=${TARGETOS} GOARCH=${TARGETARCH} \
```

Comment:
Am I missing something? I Don't think `TARGETOS` or `TARGETARCH` are set anywhere? So this is just setting `GOOS` and `GOARCH` to empty strings, which is the same as leaving them unset?

Reply:
Also, they're only set for the `mkdir` and not for the `go build`.

Reply:
🙄  That's what I get for copy & pasting as a quick test, finding it works, and then not trying to trim it down...